entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -225,18 +225,19 @@ class EntityService(
     /**
      * @return a pair consisting of a map representing the entity keys and attributes and the list of contexts
      * associated to the entity
+     * @param includeSysAttrs true if createdAt and modifiedAt have to be displayed in the entity
      */
-    fun getFullEntityById(entityId: String): JsonLdEntity? {
+    fun getFullEntityById(entityId: String, includeSysAttrs: Boolean): JsonLdEntity? {
```

Comment:
If system attributes are not returned by default, can we make includeSysAttrs false by default ?

Reply:
OK